doc/src/docs/InputOutputReference/01c-InputOutputReference.md

```diff
@@ -14192,6 +14582,25 @@ Alternate Fuel types (e.g., FuelType = NaturalGas):
 
 
 
+For the pysics based VRF model (VRF-FluidTCtrl) only:
+
+* VRF Heat Pump Indoor Unit Evaporating Temperature at Cooling Mode [C]
+
+* VRF Heat Pump Outdoor Unit Condensing Temperature at Cooling Mode [C]
+
+* VRF Heat Pump Indoor Unit Condensing Temperature at Heating Mode [C]
+
+* VRF Heat Pump Outdoor Unit Evaporating Temperature at Heating Mode [C]
+
```

Comment:
If you know the evaporating temperature, why are you using the same defrost model? I know the answer, there are no models. Just pointing out the question.